rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMEvent.java

```diff
@@ -136,4 +145,10 @@ public String toString() {
     public Object clone() throws CloneNotSupportedException {
         return super.clone();
     }
+
+    @Override
+    public int compareTo(RMEvent event) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1210)